### DISCUSSION TITLE
will renovate expose a option that can force replace already existed issue assigneer

### DISCUSSION THREAD

**Discussion by joebnb** [renovatebot/renovate#32671] [Suggest an Idea]
### Tell us more.

if one issue created and not closed,when other one do renovate check and want issue assign to him,that not work renovate only change exist issue content not include assigneer.if there have a option to force change the assigneer would be greate

**Comment by HonkingGoose** [comment]
Some things I found in our docs:

> When automerge is enabled on a PR, Renovate will _not_ add assignees or reviewers at PR creation time, in order to decrease notifications noise a little. If tests subsequently _fail_, making automerge not possible, then Renovate will add the configured assignees and/or reviewers.

[Renovate docs, assignees and reviewers](https://docs.renovatebot.com/key-concepts/automerge/#assignees-and-reviewers)

If you want, you can configure  Renovate to _always_ assign a reviewer, so even to _automerged_ PRs. Read the [Renovate docs, `assignAutomerge`](https://docs.renovatebot.com/configuration-options/#assignautomerge) to learn how.

Or maybe Renovate can only add a reviewer to a PR when it _creates_ the PR? And the problem is that Renovate can't edit the PR afterwards to add the reviewer?
